src/install/index.js

```diff
@@ -29,18 +29,38 @@ const downloadBuild = deployDoc => {
       };
       delete deployable._rev;
 
-      return DB.app.put(deployable)
-        .then(result => {
+      return utils.update(deployable)
+        .then(() => {
           debug(`Staged as ${deployable._id}`);
-          deployable._rev = result.rev;
-
           return deployable;
         });
     });
 };
 
+const writeDdocsIndividually = compiledDocs => {
```

Comment:
I think it would be more clear if this function were called `writeDdocsInSeries`. It sucks that this is needed.